crates/evm/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+//! Traits for configuring an EVM specifics.
+
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
+    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
+    issue_tracker_base_url = "https://github.com/paradigmxyz/reth/issues/"
+)]
+#![cfg_attr(not(test), warn(unused_crate_dependencies))]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+
+use reth_primitives::{revm::env::fill_block_env, Address, ChainSpec, Header, Transaction, U256};
+use revm::{Database, Evm, EvmBuilder};
+use revm_primitives::{BlockEnv, CfgEnvWithHandlerCfg, SpecId, TxEnv};
+
+/// Trait for configuring the EVM for executing full blocks.
+pub trait ConfigureEvm: ConfigureEvmEnv {
```

Comment:
assuming these will be removed from node-api in further PRs?

Reply:
good point, I'll do it in here